diff --git a/package/vl-ui-hero-navigation/src/vue/subcomponents/vl-hero-navigation-item.vue b/package/vl-ui-hero-navigation/src/vue/subcomponents/vl-hero-navigation-item.vue
index 5fc8f0b56..3c6aa1a9b 100644
--- a/package/vl-ui-hero-navigation/src/vue/subcomponents/vl-hero-navigation-item.vue
+++ b/package/vl-ui-hero-navigation/src/vue/subcomponents/vl-hero-navigation-item.vue
@@ -2,16 +2,16 @@
   <li
     :class="classes"
     v-vl-equal-height
+    @mouseenter="activeItem"
   >
-    <a v-bind="$attrs"
-      class="vl-hero-navigation__block"
-      @mouseenter="activeItem">
+    <vl-link v-bind="$attrs"
+      class="vl-hero-navigation__block">
       <h2 class="vl-hero-navigation__title">{{ title }}</h2>
       <p class="vl-hero-navigation__description">
         <slot />
       </p>
       <vl-link mod-icon-after mod-block icon="arrow-right-fat" class="vl-hero-navigation__cta">{{ linkCta }}</vl-link>
-    </a>
+    </vl-link>
   </li>
 </template>
 <script src="./vl-hero-navigation-item.js"></script>
