diff --git a/package/vl-ui-agenda-item/src/vue/vl-agenda-item.vue b/package/vl-ui-agenda-item/src/vue/vl-agenda-item.vue
index 2a9888fcf..f3b5a1bbe 100644
--- a/package/vl-ui-agenda-item/src/vue/vl-agenda-item.vue
+++ b/package/vl-ui-agenda-item/src/vue/vl-agenda-item.vue
@@ -1,7 +1,8 @@
 <template>
   <component
     :is="tagName"
-    :class="classes">
+    :class="classes"
+     v-bind="$attrs">
     <div
       itemscope
       itemtype="http://schema.org/Event">
