diff --git a/package/vl-ui-doormat/src/vue/vl-doormat.vue b/package/vl-ui-doormat/src/vue/vl-doormat.vue
index 831ffd68d..178cd4b33 100644
--- a/package/vl-ui-doormat/src/vue/vl-doormat.vue
+++ b/package/vl-ui-doormat/src/vue/vl-doormat.vue
@@ -1,7 +1,8 @@
 <template>
   <component
     :is="tagName"
-    :class="classes">
+    :class="classes"
+    v-bind="$attrs">
     <div
     v-if="image && !isInlineGraphic"
     class="vl-doormat__graphic-wrapper">
